evalml/pipelines/time_series_classification_pipelines.py

```diff
@@ -83,7 +83,11 @@ def fit(self, X, y):
 
         y_shifted = y.shift(-self.gap)
         X_t, y_shifted = drop_rows_with_nans(X_t, y_shifted)
-        self.estimator.fit(X_t, y_shifted)
+
+        if self.estimator is not None:
+            self.estimator.fit(X_t, y_shifted)
+        else:
+            self.component_graph.get_last_component().fit(X_t, y)
```

Comment:
If this pipeline does not have an estimator, this pipeline is just a series of transformations. As such, just using the original, unshifted `y` here